manifests/plugin/hdm.pp

```diff
@@ -0,0 +1,37 @@
+# = Foreman Proxy HDM plugin
+#
+# This class installs the HDM plugin
+#
+# === Parameters:
+#
+# === Advanced parameters:
+#
+# $hdm_url::                  the API endpoint of your HDM installation
```

Comment:
Since it's a mandatory parameter, I wouldn't call this advanced. Same with user and password.

Reply:
Adding to that: the regular parameters group shows up under `foreman-installer --help` while the advanced group only shows up with `--full-help`.

Reply:
done